.stylelintrc

```diff
@@ -5,7 +5,6 @@
   ],
   "defaultSeverity": "warning",
   "rules": {
-    "indentation": 2,
```

Comment:
the default for web projects used to be 2. I am not sure how strict is this rule these days but I would keep it unless there is a good reason not too